netflix-roulette/src/pages/home/components/Catalog/components/MoviesList/MovieCard/MovieCard.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+import { Movie } from "../../../../../../../utils";
+import {
+  MovieCardContainer,
+  MovieDescription,
+  MovieGenres,
+  MovieYear,
+  TitleRow,
+} from "./MovieCard.styles";
+
+interface MovieCardProps {
+  movie: Movie;
+}
+
+export const MovieCard = ({ movie }: MovieCardProps) => {
+  const { title, genres, release_date, poster_path } = movie;
+  const year = new Date(release_date).getFullYear();
```

Comment:
Parsing arbitrary date string with `new Date()` constructor is [generally discouraged](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date#parameters), so it might be safer to use your own custom parser, e.g. for `'yyyy-mm-dd'` format:
```ts
const dateStrToDate = (dateStr: string) => {
   const [yyyy, mm, dd] = dateStr.split('-')
   return new Date(
      yyyy,
      mm-1,
      dd
   )
}
```